agave_app/ViewToolbar.cpp

```diff
@@ -119,6 +96,35 @@ ViewToolbar::ViewToolbar(QWidget* parent)
   axisHelperButton->setFocusPolicy(Qt::NoFocus);
   toolbarLayout->addWidget(axisHelperButton);
 
+  toolbarLayout->addItem(new QSpacerItem(spacing, 0, QSizePolicy::Fixed, QSizePolicy::Expanding));
+
+  QMenu* menu = new QMenu("QuickViews", this);
+  menu->setObjectName("quickViewsMenu");
+  menu->setAttribute(Qt::WA_TranslucentBackground);
+  QToolBar* toolbar = new QToolBar();
+  toolbar->setObjectName("quickViewsToolbar");
+  toolbar->addWidget(rightViewButton);
+  toolbar->addWidget(leftViewButton);
+  toolbar->addWidget(frontViewButton);
+  toolbar->addWidget(backViewButton);
+  toolbar->addWidget(topViewButton);
+  toolbar->addWidget(bottomViewButton);
+  QWidgetAction* act = new QWidgetAction(toolbar);
+  act->setDefaultWidget(toolbar);
+  menu->addAction(act);
+
+  axisViewButton = new QPushButton(this);
+  axisViewButton->setObjectName("anyViewBtn");
+  axisViewButton->setToolTip(QString("<FONT>Quick Views</FONT>"));
+  axisViewButton->setStatusTip(tr("Quickly set an axis-aligned view"));
+  axisViewButton->setSizePolicy(QSizePolicy::Minimum, QSizePolicy::Expanding);
+  axisViewButton->adjustSize();
+  axisViewButton->setFocusPolicy(Qt::NoFocus);
+  toolbarLayout->addWidget(axisViewButton);
+  connect(axisViewButton, &QPushButton::clicked, [menu, this]() {
+    menu->exec(axisViewButton->mapToGlobal(axisViewButton->rect().bottomLeft() + QPoint(0, 4)));
```

Comment:
Just a knowledge question, this toggles the menu on click, right? I'm seeing that it's positioning the menu under the button, but how does it make the menu disappear once it's clicked again?

Reply:
Qt does this automatically somehow.  The behavior is: if you pop open the menu, it stays open if you click on the contained toolbar buttons.  If you click away off the menu, then it disappears.